src/System.Data.SqlClient/src/System/Data/SqlClient/SqlParameter.cs

```diff
@@ -112,6 +112,31 @@ public SqlParameter(string parameterName, SqlDbType dbType, int size, string sou
 
         public SqlParameter(
             string parameterName,
+            SqlDbType dbType,
+            int size,
+            ParameterDirection direction,
+            bool isNullable,
+            byte precision,
+            byte scale,
+            string sourceColumn,
+            DataRowVersion sourceVersion,
+            object value
+        ) : this()
```

Comment:
prefer using the constructor overload which already handles four parameters: `this(parameterName, dbType, size, sourceColumn)`

Reply:
Agree with you on this, but I figured I'd go off the reference source. I've updated the PR to use that constructor instead.